examples/remotebuildexecution/rbe_system_check/.bazelrc.sample

```diff
@@ -0,0 +1,42 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
Could you check if the internal bazelrc validation tests continue to work with this?

Reply:
We're switching to a different set of tests that work with this.